utils/dhcpdiscover/src/Makefile

```diff
@@ -0,0 +1,13 @@
+CC       = gcc
+CPPFLAGS = 
+CFLAGS   = -Os -Wall
+LDFLAGS	 =
+CONFIGURE_ARGS =
+
```

Comment:
These variables should be in the form:  xxFLAGS?=... or xxFLAGS+=... (or CC?=gcc)
Do not explicitly override build symbols.